tf2_ros/doc/index.rst

```diff
@@ -3,14 +3,51 @@ tf2_ros Overview
 
 This is the Python API reference for the tf2_ros package.
 
-To broadcast transforms using ROS:
+To broadcast transforms using ROS
+------------------------------------------------
+*Adding a TransformBroadcaster to an existing Python node*
+
 - Call :meth:`rospy.init` to initialize a node.
 - Construct a :class:`tf2_ros.TransformBroadcaster`.
 - Pass a :class:`geometry_msgs.TransformStamped` message to :meth:`tf2_ros.TransformBroadcaster.sendTransform`.
 
     - Alternatively, pass a vector of :class:`geometry_msgs.TransformStamped` messages.
 
-To listen for transforms using ROS:
+*Launching a dedicated static_transform_publisher node*
+
+A node dedicated to publish a static transformation can be launched directly from a launch file using the following syntax.
+
+In the launch file:
+
+| <node pkg="tf2_ros" type="static_transform_publisher" name="publisher_name" args="tf_frame_to_be_published_parameter_name"/>
+
+
+The tf_frame_to_be_published_parameter_name can be defined in a yaml file using the following syntax:
+
+- RPY
+    | tf_frame_to_be_published_parameter_name/header/frame_id: "parent_frame_id"
+    | tf_frame_to_be_published_parameter_name/child_frame_id: "child_frame_id"
+    | tf_frame_to_be_published_parameter_name/transform/translation/x: 0.0
+    | tf_frame_to_be_published_parameter_name/transform/translation/y: 0.0
+    | tf_frame_to_be_published_parameter_name/transform/translation/z: 0.0
+    | tf_frame_to_be_published_parameter_name/transform/rotation/R: deg(0.0)
```

Comment:
It is in radians, but making use of the more readable degree notation and the degree to radian conversion option: https://github.com/ros/ros_comm/blob/247459207e20c1da109fc306e58b84d15c4107bd/tools/rosparam/src/rosparam/__init__.py#L116